feature/s3/transfermanager/api_op_GetObject.go

```diff
@@ -0,0 +1,1106 @@
+	//
+	// [Checking object integrity]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html
+	ChecksumCRC32C string
+
+	// The base64-encoded, 160-bit SHA-1 digest of the object. This will only be
+	// present if it was uploaded with the object. For more information, see [Checking object integrity]in the
+	// Amazon S3 User Guide.
+	//
+	// [Checking object integrity]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html
+	ChecksumSHA1 string
+
+	// The base64-encoded, 256-bit SHA-256 digest of the object. This will only be
+	// present if it was uploaded with the object. For more information, see [Checking object integrity]in the
+	// Amazon S3 User Guide.
+	//
+	// [Checking object integrity]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html
+	ChecksumSHA256 string
+
+	// Specifies presentational information for the object.
+	ContentDisposition string
+
+	// Indicates what content encodings have been applied to the object and thus what
+	// decoding mechanisms must be applied to obtain the media-type referenced by the
+	// Content-Type header field.
+	ContentEncoding string
+
+	// The language the content is in.
+	ContentLanguage string
+
+	// Size of the body in bytes.
+	ContentLength int64
+
+	// The portion of the object returned in the response.
+	ContentRange string
+
+	// A standard MIME type describing the format of the object data.
+	ContentType string
+
+	// Indicates whether the object retrieved was (true) or was not (false) a Delete
+	// Marker. If false, this response header does not appear in the response.
+	//
+	//   - If the current version of the object is a delete marker, Amazon S3 behaves
+	//   as if the object was deleted and includes x-amz-delete-marker: true in the
+	//   response.
+	//
+	//   - If the specified version in the request is a delete marker, the response
+	//   returns a 405 Method Not Allowed error and the Last-Modified: timestamp
+	//   response header.
+	DeleteMarker bool
+
+	// An entity tag (ETag) is an opaque identifier assigned by a web server to a
+	// specific version of a resource found at a URL.
+	ETag string
+
+	// If the object expiration is configured (see [PutBucketLifecycleConfiguration]PutBucketLifecycleConfiguration ),
+	// the response includes this header. It includes the expiry-date and rule-id
+	// key-value pairs providing object expiration information. The value of the
+	// rule-id is URL-encoded.
+	//
+	// This functionality is not supported for directory buckets.
+	//
+	// [PutBucketLifecycleConfiguration]: https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketLifecycleConfiguration.html
+	Expiration string
+
+	// The date and time at which the object is no longer cacheable.
+	//
+	// Deprecated: This field is handled inconsistently across AWS SDKs. Prefer using
+	// the ExpiresString field which contains the unparsed value from the service
+	// response.
+	Expires time.Time
```

Comment:
We don't need to address this now but I question whether we should have the `time.Time` version of this in the transfermanager. We should be using these APIs as an opportunity to get rid of old baggage like this.